### PR TITLE
Videos list

### PR DESCRIPTION
**Pull request by Naveenchandar**
* Videos List
* Video Component
* Chips Component
* Filter videos based on chips selection
screencapture-localhost-3000-explore-2022-04-24-01_44_23 (2)

### CODE REVIEW

File: src/components/navbar/index.css [hunk 1]
```diff
@@ -4,4 +4,12 @@
     min-height: 7vh;
     padding: 1.5rem 3rem;
     font-size: 1.4rem;
+}
+
+.auth_btn {
+    color: var(--black) !important;
```
Comment (surbhi84) [inline src/components/navbar/index.css:10]: what is the purpose of using !important here ?

File: src/components/sidebar/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    return (
+        <aside className='sidebar'>
+            <ul className='mt-1 p-2 sidebar_list'>
+                <li><Link to='/' className='p-1 sidebar_link'>Home</Link></li>
+                <li><Link to='/explore' className='p-1 sidebar_link'>Explore</Link></li>
+                <li><Link to='/' className='p-1 sidebar_link'>Playlists</Link></li>
+                <li><Link to='/' className='p-1 sidebar_link'>Watch later</Link></li>
+                <li><Link to='/' className='p-1 sidebar_link'>Liked videos</Link></li>
+                <li><Link to='/' className='p-1 sidebar_link'>History</Link></li>
+            </ul>
```
Comment (surbhi84) [inline src/components/sidebar/index.js:22]: suggestion- you can use navlinks here and use activeClassName attribute for active links.

File: src/components/videos/index.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import axios from 'axios';
+import { useReducer } from 'react';
+import { useEffect } from 'react/cjs/react.development';
```
Comment (surbhi84) [inline src/components/videos/index.js]: I think only 'react' should be there, and you can import both together

**Review by surbhi84 — APPROVED** [review]
- Good use of react components.
- Nice folder structure.
- The code looks mergeable.

File: src/components/chips/index.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './index.css';
+
+const chipValues = ['all', 'rm', 'mu', 'ju', 'bm'];
```
Comment (kumsomi) [inline src/components/chips/index.js:4]: Any specific meaning of these abbreviations?

**Review by kumsomi — COMMENTED** [review]
- Just minor changes with !important. 
- Rest of the code looks good with proper folder structure.

File: src/components/videos/index.js (hunk 3 above)
Comment (Naveenchandar) [reply src/components/videos/index.js]: It was auto imported. I will update that.

File: src/components/chips/index.js (hunk 4 above)
Comment (Naveenchandar) [reply src/components/chips/index.js:4]: Yes for abbreviations for each categories.

File: src/components/sidebar/index.js (hunk 2 above)
Comment (Naveenchandar) [reply src/components/sidebar/index.js:22]: Sure. Let me change that.

File: src/components/navbar/index.css (hunk 1 above)
Comment (Naveenchandar) [reply src/components/navbar/index.css:10]: To override link styles.

**Review by kumsomi — APPROVED** [review]
Looks good to merge now.

**Pull request merged**